# tfs-ruby-3-3-x-ext-setup-nodynamic.patch
#
# Target:  ext/Setup (pass1 (all platforms))
# Applies: ruby 3.3.x (3.3.3 - 3.3.7)
# Source:  tebako gem lib/tebako/packager/pass1_patch.rb Pass1Patch::EXT_SETUP_PATCH
#
# Disable dynamic extensions: enable 'option nodynamic' so extensions are
# built statically.
diff --git a/ext/Setup b/ext/Setup
index 9c3e2f4..2d90681 100644
--- a/ext/Setup
+++ b/ext/Setup
@@ -1,4 +1,4 @@
-#option nodynamic
+option nodynamic
 
 #bigdecimal
 #cgi/escape
